packages/language/src/preprocessor/instruction-interpreter.ts

```diff
@@ -1357,21 +1346,6 @@ function resolveIncludeFileUri(
       context.currentUri,
     );
   }
-  const ext = UriUtils.extname(URI.parse(item.fileName));
-  if (
-    ext !== "" &&
-    pgroup &&
-    (!pgroup["include-extensions"]?.includes(ext) ||
-      !pgroup["include-extensions"])
-  ) {
-    const msg = pgroup["include-extensions"]?.length
-      ? `expected one of: ${pgroup["include-extensions"]?.join(", ")}`
-      : `expected no extension`;
-    throw new PreprocessorError(
-      `Unsupported include extension for included file, '${item.fileName}', ${msg}`,
-      item.token,
-    );
-  }
```

Comment:
Note that the compiler will simply emit a normal "file cannot be opened" error in this case. We do the same now.